linter_exclusions.yml

```diff
@@ -13,3 +13,10 @@ dt endpoint create eventhub:
     eventhub_resource_group:
       rule_exclusions:
       - parameter_should_not_end_in_resource_group
+central app cde create:
```

Comment:
It's the update command that is specifically giving the command table linter issue.